media/css/firefox/mobile/mobile.scss

```diff
@@ -455,17 +533,15 @@ main .mzp-l-content {
 
 @media #{$mq-sm} {
     .mzp-c-modal-window {
-        padding-left: $layout-xs;
-        padding-right: $layout-xs;
+        padding: 0 $layout-xs;
```

Comment:
Is removing the top padding on the default modal intended here?

Reply:
whoops, it is not.